### PR TITLE
Np 27816 cristin import interview media contribution

### PR DESCRIPTION
**Pull request by anetteOlli**

### CODE REVIEW

**Review by truhacevkir — APPROVED** [review]
ser bra ut

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMediumTypeCode.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        this.value = value;
+    }
+
+    public static CristinMediumTypeCode fromValue(String value) {
+        for (CristinMediumTypeCode type : CristinMediumTypeCode.values()) {
+            if (type.getValue().equalsIgnoreCase(value)) {
+                return type;
+            }
+        }
+        throw new IllegalArgumentException();
```
Comment (brinxmat) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMediumTypeCode.java:26]: Generally, I'd use a stream of values() and SingletonCollector, but I don't think it's important

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/PublicationInstanceBuilderImpl.java [hunk 2]
```diff
@@ -36,7 +37,10 @@ public PublicationInstance<? extends Pages> build() {
             return new ChapterArticleBuilder(cristinObject).build();
         } else if (isEvent(cristinObject)) {
             return new EventBuilder(cristinObject).build();
-        } else if (cristinObject.getMainCategory().isUnknownCategory()) {
+        }else if (isMediaContribution(cristinObject)) {
```
Comment (brinxmat) [inline cristin-import/src/main/java/no/unit/nva/cristin/mapper/PublicationInstanceBuilderImpl.java:40]: Formatting

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMediumTypeCode.java (hunk 1 above)
Comment (anetteOlli) [reply cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMediumTypeCode.java]: Fikset

File: cristin-import/src/main/java/no/unit/nva/cristin/mapper/PublicationInstanceBuilderImpl.java (hunk 2 above)
Comment (anetteOlli) [reply cristin-import/src/main/java/no/unit/nva/cristin/mapper/PublicationInstanceBuilderImpl.java]: fikset

**Review by brinxmat — APPROVED** [review]

**Pull request merged**